crate/kmip/src/kmip/kmip_types.rs

```diff
@@ -1093,6 +1096,54 @@ impl Attributes {
     pub fn set_object_type(&mut self, object_type: ObjectType) {
         self.object_type = Some(object_type);
     }
+
+    /// Set the attributes's CryptographicUsageMask.
+    pub fn set_cryptographic_usage_mask(&mut self, mask: Option<CryptographicUsageMask>) {
+        self.cryptographic_usage_mask = mask;
+    }
```

Comment:
I'm not sure I understand what you point out here ?

Reply:
What is the meaning of having the mask set to `None` if it is not `Unrestricted`?

My point is that both are redundant. I would either :
- have `None` to mean `Unrestricted` and no `Unrestricted` enum member;
- remove the option altogether on `self.cryptographic_usage_mask`, since the `Unrestricted` member carries enough information.